collections/_pages/glossary.html

```diff
@@ -0,0 +1,25 @@
+---
+layout:      single-section-page
+title:       "Glossary of terms"
+slug:        glossary
+intro:       "A brief overview of the most commonly used passwords and abbreviations with a short explanation or commentary."
```

Comment:
Since the glossary currently has only a single definition, can you please add a line in the intro paragraph describing that it's work in progress and more terms will be added over time?

Reply:
I've updated the paragraph with that suggestion.